docs/reference/cryptography.rst

```diff
@@ -0,0 +1,28 @@
+============
+Cryptography
+============
+
+Transport Layer Security (TLS)
```

Comment:
Shouldnt we be using sentence case for headings?

Reply:
Ah, indeed, thank you for pointing that out!